x/stakeibc/types/codec.go

```diff
@@ -45,6 +46,9 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&MsgDeleteValidator{},
 	)
+	registry.RegisterImplementations((*sdk.Msg)(nil),
```

Comment:
nit: move into above `RegisterImplementations` (we should just consolidate all of these I think)